cfy_manager/components/rabbitmq/rabbitmq.py

```diff
@@ -78,8 +82,22 @@ def _deploy_configuration(self):
 
     def _deploy_env(self):
         # This will make 'sudo rabbitmqctl' work without specifying node name
+        try:
+            nodename = config[RABBITMQ]['nodename'].split('@')[-1]
+            default_ip = config[RABBITMQ]['cluster_members'][
+                nodename]['networks']['default']
+            ipv6_enabled = network.is_ipv6(default_ip) or bool(
```

Comment:
This gives us multiple ways of determining if ipv6 is enabled. Can we perhaps move it out to its own utility function?

Reply:
Determining if IPv6 is enabled is only rabbitmq thing.  In other components there's no difference in configuration rendering between IPv4 and IPv6 environments.